### PR TITLE
Feature: Add support for TailwindCSS

### PR DESCRIPTION
**Pull request by tonyjojo**
## Description
Adds support for TailwindCSS and styles Login and Signup pages 

## Task 
[Notion Task Link](https://www.notion.so/d077118807034bbfb517f36c20d2a304?v=345833d9689145a2b5a197646677c93e&p=07798a4743bd4cd2bc009a962abe865e&pm=s)

## Screenshots

Login | Signup
-- | --
 |

### CODE REVIEW

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -1,48 +1,5 @@
 <script>
-	import { auth } from '../lib/firebase/firebase';
-	import { onMount } from 'svelte';
```
Comment (berk245) [inline src/routes/+layout.svelte:3]: Leaving a comment here to look into this. This was changed with the last PR and I am not sure if it's supposed to be removed. Wdyt?

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -1,48 +1,5 @@
...
-				// Authenticated user
-				goto(currentPath);
-				return;
-			}
-		});
-	});
+	import '../app.postcss';
 </script>
 
-{#if $authStore.user}
```
Comment (berk245) [inline src/routes/+layout.svelte:45]: Okay we shouldn't remove the sign out button, right? Unless you are introducing a new way to sign out and redirect users?

**Review by berk245 — CHANGES_REQUESTED** [review]

File: src/routes/+layout.svelte (hunk 1 above)
Comment (tonyjojo) [reply src/routes/+layout.svelte:3]: You're right, I messed up the layout for some reason, got confused, sorry

File: src/routes/+layout.svelte (hunk 2 above)
Comment (tonyjojo) [reply src/routes/+layout.svelte:45]: Re-added it

File: src/routes/dashboard/+layout.svelte [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<script>
```
Comment (berk245) [inline src/routes/dashboard/+layout.svelte]: I think we moved away from dashboard layout and moved all the sign out and rerouting logic to the main layout. @niccoloMG can you confirm?

**Review by berk245 — CHANGES_REQUESTED** [review]
Looks beautiful! Let's fix the auth and layout related issues and merge it

File: src/routes/dashboard/+layout.svelte (hunk 3 above)
Comment (niccoloMG) [reply src/routes/dashboard/+layout.svelte]: Yep. dashboard/layout file isn't needed anymore. Conditional sign out button render and rerouting happens in the main layout

**Review by berk245 — APPROVED** [review]
Looks great! Thank you

**Pull request merged**